master/security/service-accounts.md

```diff
@@ -86,6 +86,29 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Restrict label assignment with service account selectors
```

Comment:
You can apply network policies using selectors that match labels on the endpoint, the endpoint namespace, or the endpoint service account. If you specify selectors based on the endpoint service account, you can apply Kubernetes RBAC to limit which users are allowed to apply labels. In the following example, users with the **intern role** are allowed to communicate only with endpoints with the same roles.